src/webportal/src/app/job-submission/components/data/team-storage - Copy.jsx

```diff
@@ -0,0 +1,244 @@
+/* !
```

Comment:
This file name is strange, commit by mistake?

Reply:
this is a backup file, I will delete it.